src/figuresClasses.ts

```diff
@@ -1,11 +1,96 @@
-export interface Figure {}
+// Interface
+export interface Figure {
+  shape: string;
```

Comment:
```suggestion
  shape: Shape;
```
Consider creating an enum for the shape